### ISSUE TITLE
Songs: Auto-import tags from original to cover

### ISSUE THREAD

**Issue by Zircon777** [VocaDB/vocadb#1329]
Pretty straight forward, tags being automatically added when creating a cover entry, some originals have a lot of tags and this would make everyone's life more easy

**Comment by szc126** [comment]
For situations where tags should not apply to the child entry, (if the tags are to be synced at all times,) perhaps it could also be resolved using a tag votes system; tags that do not belong on the child entry can be marked as inappropriate using negative votes.

**Comment by CatgirlFrostmoon** [comment]
Honestly, I think this would cause more problems then it's worth. While the average cover may be a relatively one-to-one version, not everything we have is an "average cover". Having to remove all of these would take quite a long time, especially since people don't always track auto-tags.

**Genre + Instruments + Composition** : remixes are often placed under "cover" by people who add entries without listening to the songs, plus covers sometimes use out-of-scope instrumentals
**Themes + Subjective**: should mostly work, but parodies can and will screw with this
**Animations + MMD Models**: complete no go, as many covers use their own images; this includes the epilepsy warning tag, too
**Copyrights**: I...genuinely can't think of something where an originating copyright wouldn't apply, so this should be fine...?
**Lyrics**: anything that changes the lyrics could mess with this
All other tag groups are definitely ones that should stay as a purposeful add to a song, as they refer to things that are way to idiosyncratic for inheritance to work.

In other words, the only one without serious issues is the Copyright tag group.
